packages/base/src/mainview/mainView.tsx

```diff
@@ -142,6 +142,17 @@ export class MainView extends React.Component<IProps, IStates> {
       this
     );
 
+    this._model.highlightFeatureOnMap = this.highlightFeatureOnMap.bind(this);
+    this._model.flyToGeometry = this.flyToGeometry.bind(this);
```

Comment:
😁 This is a code smell to me. If you open the same document twice, I guess this logic breaks?